spec/integration/indirector/trusted_external/exec_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+require 'puppet_spec/files'
+require 'puppet_spec/compiler'
+require 'puppet/indirector/trusted_external/exec'
+
+describe Puppet::TrustedExternal::Exec do
+  include PuppetSpec::Files
+  include PuppetSpec::Compiler
+
+  context "with a trusted_external_command configured" do
+    let(:script_path) do
+      script_containing('script',
+        windows: "@echo '{ \"fact\": \"ABCD\" }'"
```

Comment:
Single quotes are *NOT* a string delimiter in `cmd.exe`

In fact, echo will literally spit out the text anyway

```
C:\Users\glenn.sarti>@echo { "fact": "ABCD" }
{ "fact": "ABCD" }

C:\Users\glenn.sarti>@echo '{ "fact": "ABCD" }'
'{ "fact": "ABCD" }'
```